src/qml/filters/deband/ui.qml

```diff
@@ -0,0 +1,440 @@
+            filter.set(blurParam, true)
+            filter.set(couplingParam, false)
+            filter.savePreset(allParams, qsTr('Full range to Limited range'))
+
+            // Default preset
+            // Same as Full to Limited preset
+            filter.savePreset(allParams)
+        }
+        filter.blockSignals = false
+
+        setControls()
+    }
+
+
+    width: 350
+    height: 450
+
+
+    GridLayout {
+        columns: 3
+        anchors.fill: parent
+        anchors.margins: 8
+
+        // Row split
+
+        Label {
+            text: qsTr('Preset')
+            Layout.alignment: Qt.AlignRight
+        }
+        Shotcut.Preset {
+            id: idPreset
+            Layout.columnSpan: 2
+            parameters: allParams
+            onPresetSelected: setControls()
+        }
+
+        // Row split
+
+        Label {
+            text: qsTr('Y/R Threshold')
+            Layout.alignment: Qt.AlignRight
+        }
+        Shotcut.SliderSpinner {
+            id: idThr1
+            minimumValue: 0
+            maximumValue: 100
+            decimals: 1
+            suffix: ' %'
+            onValueChanged: setThreshold(thr1Param, value)
+        }
+        Shotcut.UndoButton {
+            onClicked: idThr1.value = thrToPct(thr1Default)
+        }
+
+        // Row split
+
+        Label {
+            text: qsTr('Cb/G Threshold')
+            Layout.alignment: Qt.AlignRight
+        }
+        Shotcut.SliderSpinner {
+            id: idThr2
+            minimumValue: 0
+            maximumValue: 100
+            decimals: 1
+            suffix: ' %'
+            onValueChanged: setThreshold(thr2Param, value)
+        }
+        Shotcut.UndoButton {
+            onClicked: idThr2.value = thrToPct(thr2Default)
+        }
+
+        // Row split
+
+        Label {
+            text: qsTr('Cr/B Threshold')
+            Layout.alignment: Qt.AlignRight
+        }
+        Shotcut.SliderSpinner {
+            id: idThr3
+            minimumValue: 0
+            maximumValue: 100
+            decimals: 1
+            suffix: ' %'
+            onValueChanged: setThreshold(thr3Param, value)
+        }
+        Shotcut.UndoButton {
+            onClicked: idThr3.value = thrToPct(thr3Default)
+        }
+
+        // Row split
+
+        Label {
+            text: qsTr('Alpha Threshold')
+            Layout.alignment: Qt.AlignRight
+        }
+        Shotcut.SliderSpinner {
+            id: idThr4
+            minimumValue: 0
+            maximumValue: 100
+            decimals: 1
+            suffix: ' %'
+            onValueChanged: setThreshold(thr4Param, value)
+        }
+        Shotcut.UndoButton {
+            onClicked: idThr4.value = thrToPct(thr4Default)
+        }
+
+        // Row split
+
+        Label {}
+        CheckBox {
+            id: idLink
+            text: qsTr('Link thresholds')
+            onClicked: filter.set(linkParam, checked)
+        }
+        Label {}
```

Comment:
You can either use an Item{} here or set columnspan: 2 on idLink
https://doc.qt.io/qt-5/qml-qtquick-layouts-layout.html#columnSpan-attached-prop

Reply:
The idea here was to prevent the checkbox text from spilling into the Undo column, which could create a ragged-edge appearance with long translated strings.  If Item{} is a suitable replacement, should I also use Item{} in place of the Label{} that's in the first column of the checkbox rows?

Reply:
Item is typically used if you just need a filler in a cell. If there is some other reason to use Label, that is fine, too.

Reply:
Item {} caused layout issues due to collapsing to zero width.
I had to use Item { Layout.fillWidth: true } which hopefully is okay.
I replaced all occurrences of Label {} with Item.